containers/kubernetes/workloads/explorer-jes-deployment.yaml

```diff
@@ -61,11 +61,11 @@ spec:
               cpu: "100m"
           ports:
             - name: jes-port
-              containerPort: 8546
```

Comment:
Unsure what happens with `containers/kubernetes/workloads/explorer*` files in v2.
Potentially they still exist but without ports.
Or they may be removed.